pkg/webservice/dao/application_info.go

```diff
@@ -48,4 +48,5 @@ type PlaceholderDAOInfo struct {
 	Count         int64            `json:"count"`
 	MinResource   map[string]int64 `json:"minResource"`
 	Replaced      int64            `json:"replaced"`
+	Timeout       int64            `json:"timeout"`
```

Comment:
Missing "d" in the name: Time**d**Out Also capitalise the `O`
json field should be timedout also.

Reply:
Done.Thanks.